.readme-partials.yaml

```diff
@@ -46,6 +48,106 @@ custom_content: |
    
+   ### OpenTelemetry Tracing
+   The Spanner JDBC driver supports OpenTelemetry tracing. You can configure the OpenTelemetry instance
+   that should be used in two ways:
+   1. Register a global OpenTelemetry instance. This instance will automatically be picked up by the Spanner JDBC driver.
+   2. Add an OpenTelemetry instance with the key `openTelemetry` to the `java.util.Properties` instance that is used to create the JDBC connection.
+   
+   By default, the traces that are generated by the Spanner JDBC driver do not include the SQL
+   statement. You can include the SQL statement with the traces by adding the property `enableExtendedTracing=true`
+   to the JDBC connection URL.
+   
+   #### Example Using Global OpenTelemetry
+   Create and register a global OpenTelemetry object before creating a JDBC connection.
+   See also the [Spring Data JDBC Sample](samples/spring-data-jdbc) for an example for how to
+   configure OpenTelemetry in combination with Spring Data.
+   
+   See [Latency Debugging Guide](documentation/latency-debugging-guide.md) for more information on how to use these traces.
+
+   ```java
+   TraceConfiguration traceConfiguration = TraceConfiguration.builder().setProjectId("my-project").build();
+   SpanExporter traceExporter = TraceExporter.createWithConfiguration(traceConfiguration);
+   OpenTelemetry openTelemetry =
+        OpenTelemetrySdk.builder()
+            .setTracerProvider(
+                SdkTracerProvider.builder()
+                    .setSampler(Sampler.traceIdRatioBased(0.05))
+                    .setResource(
+                        Resource.builder()
+                            .put("service.name", "my-unique-service-name")
+                            .build())
+                    .addSpanProcessor(BatchSpanProcessor.builder(traceExporter).build())
+                    .build())
+            .buildAndRegisterGlobal();
+   String projectId = "my-project";
+   String instanceId = "my-instance";
+   String databaseId = "my-database";
+   // Setting this to true instructs the JDBC driver to include the SQL statement with the traces.
+   boolean enableExtendedTracing = true;
+   
+   try (Connection connection =
+       DriverManager.getConnection(
+           String.format(
+               "jdbc:cloudspanner:/projects/%s/instances/%s/databases/%s?enableExtendedTracing=%s",
+               projectId, instanceId, databaseId, enableExtendedTracing))) {
+     try (Statement statement = connection.createStatement()) {
+       try (ResultSet rs = statement.executeQuery("SELECT CURRENT_TIMESTAMP()")) {
+         while (rs.next()) {
+           System.out.printf(
+               "Connected to Cloud Spanner at [%s]%n", rs.getTimestamp(1).toString());
+         }
+       }
+     }
+   }
+   ```
+   
+   #### Example Using an OpenTelemetry instance in Properties
+   Create and register a global OpenTelemetry object before creating a JDBC connection.
```

Comment:
We are not registering global OT object here. 

nit: I feel we can simplify this example by not repeating the whole code . 

We can create OpenTelemetrySdkBuilder object and then show different usage of it. 

for eg. 

```
 OpenTelemetrySdkBuilder openTelemetrySdkBuilder =
        OpenTelemetrySdk.builder()
            .setTracerProvider(
                SdkTracerProvider.builder()
                    .setSampler(Sampler.traceIdRatioBased(0.05))
                    .setResource(
                        Resource.builder()
                            .put("service.name", "my-unique-service-name")
                            .build())
                    .addSpanProcessor(BatchSpanProcessor.builder(traceExporter).build())
                    .build());
 // Example Using Global OpenTelemetry
 openTelemetrySdkBuilder.buildAndRegisterGlobal()
// Example Using an OpenTelemetry instance in Properties 
     OpenTelemetry openTelemetry = openTelemetrySdkBuilder.build()
  ```

Reply:
Thanks for noting. Removed the text regarding registering a global OTEL object.

I also simplified the second example a bit, assuming that readers will understand that they can use the same OpenTelemetry instance as in the example above.